src/components/v5/common/ActionSidebar/partials/CompletedAction/CompletedActionSidebar.tsx

```diff
@@ -0,0 +1,314 @@
+              )
+            }
+          />
+        </div>
+      </ActionSidebarLayout>
+    );
+  }
+
+  const actionType = getExtendedActionType(action, colony.metadata);
+
+  const getActionContent = () => {
+    switch (actionType) {
+      case ColonyActionType.Payment:
+      case ColonyActionType.PaymentMotion:
+        return <SimplePayment action={action} />;
+      case ColonyActionType.MintTokens:
+      case ColonyActionType.MintTokensMotion:
+        return <MintTokens action={action} />;
+      case ColonyActionType.MoveFunds:
+      case ColonyActionType.MoveFundsMotion:
+        return <TransferFunds action={action} />;
+      case ColonyActionType.CreateDomain:
+      case ColonyActionType.CreateDomainMotion:
+      case ColonyActionType.EditDomain:
+      case ColonyActionType.EditDomainMotion:
+        return <ManageTeam action={action} />;
+      case ColonyActionType.UnlockToken:
+      case ColonyActionType.UnlockTokenMotion:
+        return <UnlockToken action={action} />;
+      case ColonyActionType.VersionUpgrade:
+      case ColonyActionType.VersionUpgradeMotion:
+        return <UpgradeColonyVersion action={action} />;
+      case ColonyActionType.CreateDecisionMotion:
+        return <CreateDecision action={action} />;
+      case ColonyActionType.SetUserRoles:
+      case ColonyActionType.SetUserRolesMotion:
+        return <SetUserRoles action={action} />;
+      case ColonyActionType.AddVerifiedMembers:
+      case ColonyActionType.AddVerifiedMembersMotion:
+        return <AddVerifiedMembers action={action} />;
+      case ColonyActionType.RemoveVerifiedMembers:
+      case ColonyActionType.RemoveVerifiedMembersMotion:
+        return <RemoveVerifiedMembers action={action} />;
+      case ColonyActionType.EmitDomainReputationReward:
+      case ColonyActionType.EmitDomainReputationRewardMotion:
+      case ColonyActionType.EmitDomainReputationPenalty:
+      case ColonyActionType.EmitDomainReputationPenaltyMotion:
+        return <ManageReputation action={action} />;
+      case ColonyActionType.ColonyEdit:
+      case ColonyActionType.ColonyEditMotion:
+        return <EditColonyDetails action={action} />;
+      case ExtendedColonyActionType.UpdateColonyObjective:
+        return <UpgradeColonyObjective action={action} />;
+      // @TODO: Connect this to the reputation actions
+      /* case ColonyActionType.EmitDomainReputationReward:
+         case ColonyActionType.EmitDomainReputationPenalty:
+          return <ManageReputation action={action} />; */
+      case ColonyActionType.CreateExpenditure:
+        return <PaymentBuilder action={action} />;
+      case ColonyActionType.ManageTokens:
+      case ColonyActionType.ManageTokensMotion:
+        return <ManageTokens action={action} />;
+      default:
+        console.warn('Unsupported action display', action);
+        return <div>Not implemented yet</div>;
+    }
+  };
+
+  const getSidebarWidgetContent = () => {
+    switch (actionType) {
```

Comment:
we would need to add the reputation management motion to the switch (or turn it around if it's a motion, render the motions component, etc)
Not your change, just thinking if we can do it now while we're at it